### PR TITLE
favicon, closes #16

### LINKED ISSUE #16: favicon
Needs to have a favicon, as this isn't great:

2015-01-07 at 7 54 am

### PR DESCRIPTION
**Pull request by sparty02**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -12,6 +12,9 @@
   <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ "/feed.xml" | prepend: site.baseurl | prepend: site.url }}" />
 
+  <link rel="shortcut icon" href="{{ "/images/favicon.ico" | prepend: site.baseurl }}" type="image/x-icon">
+  <link rel="icon" href="{{ "/images/favicon.ico" | prepend: site.baseurl }}" type="image/x-icon">
```
Comment (thejameskyle) [inline _includes/head.html]: Why both of these?

Comment (sparty02) [inline _includes/head.html]: That's a great question.  I did some research on favicons and there is an obnoxious variety of ways you can import/link these things.  It reminds me of vendor-prefixed css...  :cry:   (see http://realfavicongenerator.net/ as an example).  Maybe the thing to do is to just use:
```
<link rel="icon"
```
and then layer on others if/when the need arises.

Comment (sparty02) [inline _includes/head.html]: On a related note, I couldn't figure out how to tell Jekyll to pick up files off the root ('they' recommend putting favicon.ico in the root of the site).....which is why I put it in images.  @thejameskyle do you know?

Comment (thejameskyle) [inline _includes/head.html]: @sparty02 you should just be able to put it in the root of the directory.

**Comment by sparty02** [comment]
@thejameskyle I think this should be ready to go now (put favicon in root and cleaned up multiple links in head)

**Comment by thejameskyle** [comment]
Sweet, thanks!

**Pull request merged**